docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,88 @@
+from iris.coord_systems import GeogCS
+import iris.plot as iplt
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def main():
+    # Start with arrays for latitudes and longitudes, with a given resolution.
+    resolution = 200
+    longitudes = np.linspace(-180.0, 180.0, resolution)
+    latitudes = np.linspace(-90.0, 90.0, resolution)
+    lon2d, lat2d = np.meshgrid(longitudes, latitudes)
+
+    # Omega is the Earth's rotation rate, expressed in radians per second
+    omega = 7.29e-5
+
+    # Now to create our data
+    data = 2. * omega * np.sin(np.deg2rad(lat2d))
+
+    # We now need to define a coordinate system for the plot.
+    # Here we'll use GeogCS; 6371229 is the radius of the Earth in metres.
+    cs = GeogCS(6371229)
+
+    # The Iris coords module turns the latitude list into a coordinate array.
+    # Coords then applies an appropriate standard name and unit to it.
+    lat_coord = iris.coords.DimCoord(latitudes,
+                                     standard_name='latitude',
+                                     units='degrees',
+                                     coord_system=cs)
+
+    # The above process is repeated for the longitude coordinates.
+    long_coord = iris.coords.DimCoord(longitudes,
+                                      standard_name='longitude',
+                                      units='degrees',
+                                      coord_system=cs)
+
+    # The next step is to add vertical structure to the cube.
+    # This cube has only one level, but a cube with multiple model levels
+    # can be made by feeding in a list rather than a single value.
+    height_coord = iris.coords.DimCoord(100.,
+                                        bounds=[90., 110.],
+                                        standard_name=None,
+                                        units='m',
+                                        long_name='level_height',
+                                        attributes={'positive': 'up'})
+
+    # Now we input our data array into the cube.
+    new_cube = iris.cube.Cube(data,
+                              standard_name='coriolis_parameter',
+                              units='s-1',
+                              dim_coords_and_dims=[(lat_coord, 0),
+                                                   (long_coord, 1)],
+                              aux_coords_and_dims=[(height_coord, None)])
+
+    # This section checks to ensure that the coordinates are bounded.
+    # If they have no bounds, the code applies some via guess_bounds()
+    for coord in new_cube.coords():
+        if not isinstance(coord.bounds, np.ndarray):
+            coord.guess_bounds()
+
+    # Now let's plot our cube, along with coastlines, a title and an
+    # appropriately-labelled colour bar:
+    ax = plt.axes(projection=ccrs.Orthographic())
+    ax.coastlines(resolution='10m')
+    mesh = iplt.pcolormesh(new_cube, cmap='seismic')
+    bar = plt.colorbar(mesh, orientation='horizontal')
```

Comment:
That makes sense. And I suppose labels with exponential notation is something science users would find useful, so I'll see to adopting that.